WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -385,6 +399,19 @@ ZmPreviewPaneView.prototype._listSelectionListener = function(ev, item) {
     	} else {
     		this._previewView.set(item);
     	}
+    } else if(!this._controller.isReadingPaneOn() && item && item.folderId == ZmFolder.ID_FILE_SHARED_WITH_ME){
+        var restUrl = item.getRestUrl();
+        restUrl = AjxStringUtil.fixCrossDomainReference(restUrl);
+        var req = new XMLHttpRequest();
```

Comment:
why do we want to send an extra request? aren't we sending this request when loading file data in preview pane?